tests/combinatorics/test_combinatorics.py

```diff
@@ -1,13 +1,12 @@
-import pyrival.combinatorics
+from pyrival.combinatorics import *
 
+def test_catalan_recursive(A):
+    assert all(catalan_recursive(i) == j for i, j in enumerate(A))
 
-def test_catalan_recursive(catalan):
-    assert all(pyrival.combinatorics.catalan_recursive(i) == j for i, j in enumerate(catalan))
 
+def test_derangements(A):
+    assert all(derangements(i) == j for i, j in enumerate(A))
 
-def test_derangements(derangements):
-    assert all(pyrival.combinatorics.derangements(i) == j for i, j in enumerate(derangements))
 
-
-def test_catalan(catalan):
-    assert all(pyrival.combinatorics.catalan(i) == j for i, j in enumerate(catalan))
+def test_catalan(A):
```

Comment:
Function arguments cannot be renamed since pytest passes the fixture based on function argument name

Reply:
you can change the name of the fixture functions in [tests/combinatorics/conftest.py](https://github.com/bjorn-martinsson/PyRival/blob/simplify_init/tests/combinatorics/conftest.py) to fix this